style.css

```diff
@@ -1,26 +1,45 @@
-.follow-nature li {
-  position: relative;
+.follow-nature ul {
+    overflow: visible;
+}
 
+.follow-nature li {
+    position: relative;

```

Comment:
as discussed, this is set so each `position: absolute` `<a>` element refers to its parent `<li>` element in order to work out where it should be on the page